test/service/src/lib.rs

```diff
@@ -398,7 +398,8 @@ pub struct TestNode {
 	pub transaction_pool: TransactionPool,
 }
 
-enum Consensus {
+#[allow(missing_docs)]
+pub enum Consensus {
```

Comment:
`pub(crate)` might be what you are looking for here? Then you do not need to document it either.

Reply:
Hmm yes but I am using this in a binary which can afaik only access pub items. I would need to create some other `main()` in a  and call it from the binary. Is that the usual way to do it?

Reply:
Right, sorry, I have overlooked something because you seemed to be even in the same module based on the imports. Never had 2 crates in the same folder before and thought for some reason they were the same crate. But according to the [Rust documentation](https://doc.rust-lang.org/book/ch07-01-packages-and-crates.html):

> If a package contains src/main.rs and src/lib.rs, it has two crates: a binary and a library, both with the same name as the package.

So the `lib.rs` is imported from a different crate with the same name.